pkg/config.go

```diff
@@ -108,9 +129,16 @@ func (c *ScrapeConf) Load(file *string) error {
 	return nil
 }
 
+func (c *ScrapeConf) IsConfigFileEmpty() bool {
```

Comment:
Not worth having a separate function here, maybe just add `&& c.CustomMetrics == nil` to the original check.